src/SFA.DAS.Authorization.EmployerFeatures/AuthorizationHandler.cs

```diff
@@ -32,7 +32,7 @@ public Task<AuthorizationResult> GetAuthorizationResult(IReadOnlyCollection<stri
                 {
                     authorizationResult.AddError(new EmployerFeatureNotEnabled());
                 }
-                else if (!featureToggle.IsUserEmailWhitelisted(values.UserEmail))
+                else if (featureToggle.IsWhitelistEnabled && !featureToggle.IsUserWhitelisted(values.AccountId, values.UserEmail))
```

Comment:
at the moment looks like featureToggle.IsWhitelistEnabled in a non unit test scenario will invariably evaluate to false, is this expected?